internal/internal_workflow_testsuite.go

```diff
@@ -2967,6 +3161,183 @@ func (h *testNexusOperationHandle) cancel() {
 				h.completedCallback(nil, fmt.Errorf("operation cancelation handler failed: %v", failure.GetError().GetFailure().GetMessage()))
 			}
 			h.env.runningCount--
+			if h.env.onNexusOperationCanceledListener != nil {
+				h.env.onNexusOperationCanceledListener(h.params.client.Service(), h.params.operation)
+			}
```

Comment:
I'm not sure how useful is the canceled listener here. It doesn't seem that the user can cancel a Nexus operation through the SDK.

Reply:
I think this is helpful for users to assert that an operation cancelation was requested by the workflow.